docs/reference/misc.rst

```diff
@@ -118,7 +225,7 @@ Here is an example:
       (a * b * c) ={ sym (multAssociative a b c) }=
       (a * (b * c)) ={ cong (multCommutative b c) }=
       (a * (c * b)) ={ multAssociative a c b }=
-      (a * c * b) ={ cong {f = (*b)} (multCommutative a c) }=
+      (a * c * b) ={ cong {f = (* b)} (multCommutative a c) }=
```

Comment:
Vim's reSt syntax plugin makes everything italic from this point to the end of the file. Hotfix: add space.